build.rs

```diff
@@ -0,0 +1,31 @@
+extern crate rustc_version;
+
+use rustc_version::{version_meta, Channel};
+use std::env;
+
+fn main() {
+    assert!(
+        !(env::var("CARGO_FEATURE_REVERSE_BITS").is_ok()
+            && env::var("CARGO_FEATURE_NO_REVERSE_BITS").is_ok()),
+        "Can't both enable and disable REVERSE_BITS"
+    );
+
+    // Set features depending on channel
+    match version_meta().unwrap().channel {
```

Comment:
Would this better be `if let Channel::Nightly = version_meta().unwrap().channel { ... } else { ... }`?

Reply:
done, especially since we no longer handle the feature settings.